analyzer_test.go

```diff
@@ -18,7 +18,7 @@ func TestDeriveManifestAndLock(t *testing.T) {
 	defer h.Cleanup()
 
 	h.TempDir("dep")
-	golden := "analyzer/manifest.json"
+	golden := "analyzer/manifest.toml"
```

Comment:
I realize you didn't introduce this, but now that I'm seeing it, I believe it should be `filepath.Join("analyzer", ManifestName)`.